frontend/src/metabase/lib/formatting/url.tsx

```diff
@@ -40,6 +41,12 @@ export function formatUrl(value: string, options: OptionsType = {}) {
   if (jsx && rich && url) {
     const text = getLinkText(value, options);
     const className = cx(CS.link, CS.linkWrappable);
+
+    // (metabase#51099) prevent url from being rendered as a link when in sdk
+    if (isEmbeddingSdk) {
+      return url;
+    }
```

Comment:
I removed this line and the test suite (`e2e/test-component/scenarios/embedding-sdk/interactive-dashboard.cy.spec.tsx`) still passes, that means it didn't assert this part of the code.

Reply:
@WiNloSt oh yes, I think that is because it won't format the URL in the first place because of a condition I put in https://github.com/metabase/metabase/pull/51290/files#diff-9f8b69e52e4308de8ce217916c6f07365263f6532626dc64d4ff2976f05cac69R169-R170 - so I think the tests is correct but we might not need that change in formatting/url

Reply:
let me remove that and see if the tests pass

Reply:
@WiNloSt I added another test case for columns with URL semantic type: https://github.com/metabase/metabase/pull/51290/commits/e40eca0fb4ad9055da3555c32d352a150b91dab3. This updates the ORDERS.PRODUCT_ID column to `type/URL` which should show up as clickable URL on Metabase app, but now becomes a plain column.

<img src="https://github.com/user-attachments/assets/4e4f3719-7c6f-44d2-bb6c-775a646ee108" width="400">